src/pages/help/telemetry.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Telemetry
+description:
+---
+
+<PageDescription>
+
+Carbon contains a telemetry feature that collects usage information for IBM and
+Carbon Design System properties.
+
+</PageDescription>
+
+### Why is telemetry collected?
+
+Carbon collects telemetry information to influence Carbon's roadmap and
+prioritize bug fixes.
+
+### When is data collected?
+
+Data is only collected when Carbon packages are installed on a Continuous
+Integration (CI) server, when the installation is not part of a Pull Request
+(PR), for Git respositories with the origin remotes:
```

Comment:
```suggestion
(PR), and for Git repositories with the origin remotes:
```

Reply:
@mattrosno It looks like a word was missing here. Could you pls check to make sure I've kept your intended meaning?

Reply:
Thank you.